.github/workflows/ci_test.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+    with:
+        command: bundle exec rubocop --parallel --format github
+
+  test:
+    name: Tests
+    needs: rubocop
+    uses: theforeman/actions/.github/workflows/test-gem.yml@v0
+    with:
+      command: bundle exec rake test
```

Comment:
This is not going to fly as there are not tests and therefore no rake task to run them
https://github.com/theforeman/hammer-cli-foreman-tasks/actions/runs/9956681232/job/27507158707?pr=57#step:5:6